test/1_test_fullpath.ts

```diff
@@ -404,12 +448,10 @@ describe('Voucher tests', () => {
       );
 
       //Check ERC1155ERC721 state
-      const sellerERC1155ERC721Balance = (
-        await contractERC1155ERC721.functions[fnSignatures.balanceOf1155](
-          users.seller.address,
-          tokenSupplyKey1
-        )
-      )[0];
+      const sellerERC1155ERC721Balance = await contractVoucherSets.balanceOf(
```

Comment:
Can we please rename `sellerERC1155ERC721Balance` to something without the `ERC1155ERC721` text?

Variable names with the `ERC1155ERC721` text appears in 3 files:
1. 1_test_fullpath.ts
2. 2_test_fullpath_with_permit.ts
3. testHelpers/utils.ts

Reply:
This would cause a LOT of rework in a LOT of test files and introduce possible test issues.  I don't personally think it's worth the amount of work or the risk.

Reply:
I confirm that we discussed that with and I agreed that these names can stay as they are, since it has no functional impact.
Yes it would be nicer, but not really worth doing it.

Reply:
I went ahead and changed the ones I could find, but if there are any references still left, I'm not going to spend any more time on it. Will be included in next commit